lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -218,6 +220,8 @@ class TestCohortStudents(TestReportMixin, InstructorTaskCourseTestCase):
     Tests that bulk student cohorting works.
     """
     def setUp(self):
+        super(TestCohortStudents, self).setUp()
+
```

Comment:
Is there some kind of validator we can use to check for unittest classes having a setUp() of one line or more without having a super().setUp() call?